WBooks/Screens/Library/CustomBookCell/CustomBookCellViewModel.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  CustomBookCellViewModel.swift
+//  WBooks
+//
+//  Created by Nacho 2 on 02/04/2020.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import UIKit
+
+class CustomBookCellViewModel {
+    var bookModel: Book
+    
+    init(bookModel: Book) {
+        self.bookModel = bookModel
+    }
+    
+    var image: UIImage {
+        guard let imageName = bookModel.image else { return UIImage() }
```

Comment:
you can validate the image in the guard `guard let imageName = bookModel.image , let image. = UIImage(named: imageName) else { return image }`